server/core/templates/admin/admin_partial_base.html

```diff
@@ -0,0 +1,54 @@
+<main class="container-xxl pb-5">
+    <div class="d-flex flex-sm-row flex-column">
+        <div class="list-group side-menu">
+            <a href="/ui/admin" hx-target="#main" class="list-group-item list-group-item-action (% block overview_item_extra_classes%)(%endblock%)">
```

Comment:
Why all the blocks for attributes and not just variables?

Reply:
They're used to mark the active menu-item by the concrete page by implementing the block. 
I felt that this was clearer although hacky and more cohesive than passing variables from rust